source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -0,0 +1,165 @@
+.. role:: javascript(code)
+  :language: javascript
+
+========================================
+Connection Monitoring and Pooling (CMaP)
+========================================
+
+.. contents::
+
+--------
+
+Introduction
+============
+
+The YAML and JSON files in this directory are platform-independent tests that
+drivers can use to prove their conformance to the Connection Monitoring and Pooling (CPAM) Spec.
```

Comment:
This probably should be changed to "CMAP" for clarity

Reply:
oops